src/main/scala/advancepassengerinfo/importer/processor/DqFileProcessorImpl.scala

```diff
@@ -31,13 +31,25 @@ case class DqFileProcessorImpl(manifestsProvider: Manifests, persistence: Persis
             persistence
               .persistZipFile(zipFileName, successful > 0)
               .map(_ => Option((total, successful)))
-          }
+          }.recover {
```

Comment:
Can you put the `.recover` calls on the next line so they're consistently indented with the other chained function calls? Then it's easier to read and see what's going on :+1: